backend/src/schema/types/type/REPORTED.gql

```diff
@@ -0,0 +1,42 @@
+type REPORTED {
+  createdAt: String
+  reasonCategory: ReasonCategory
+  reasonDescription: String
+  submitter: User
+    @cypher(statement: "MATCH (resource)<-[:REPORTED]-(user:User) RETURN user")
+  # not yet supported
+  # resource: ReportResource
+  #   @cypher(statement: "MATCH (resource)<-[:REPORTED]-(user:User) RETURN resource")
+  type: String
+    @cypher(statement: "MATCH (resource)<-[:REPORTED]-(user:User) RETURN labels(resource)[0]")
+  user: User
+  post: Post
+  comment: Comment
+}
+
+# this list equals the strings of an array in file "webapp/components/Modal/ReportModal.vue"
```

Comment:
This file has changed to `webapp/constants/modals.js` …
I’ll push this up now …